### PR TITLE
Refactoring due to adding Consts class

### PR DESCRIPTION
**Pull request by ashurakova**

### CODE REVIEW

File: SeaBattleLibrary/ShipPlacementValidation.cs [hunk 1]
```diff
@@ -1,38 +1,39 @@
...
-                        if (BattleShip.BotField[i + 1, x] == 1 || BattleShip.BotField[i, x - 1] == 1 || BattleShip.BotField[i + 1, x - 1] == 1)
+                        if (BattleShip.BotField[i + 1, x] == Cells.Ship || BattleShip.BotField[i, x - 1] == Cells.Ship || BattleShip.BotField[i + 1, x - 1] == Cells.Ship)
                         {
                             counter++;
                         }
                     }
                     else
                     {
-                        if (BattleShip.BotField[i, x - 1] == 1 || BattleShip.BotField[i, x + 1] == 1 || BattleShip.BotField[i + 1, x] == 1 || BattleShip.BotField[i + 1, x + 1] == 1 || BattleShip.BotField[i + 1, x - 1] == 1)
+                        if (BattleShip.BotField[i, x - 1] == 1 || BattleShip.BotField[i, x + 1] == Cells.Ship || BattleShip.BotField[i + 1, x] == Cells.Ship || BattleShip.BotField[i + 1, x + 1] == Cells.Ship || BattleShip.BotField[i + 1, x - 1] == Cells.Ship)
```
Comment (HeorhiSemianeniaiTechArt) [inline SeaBattleLibrary/ShipPlacementValidation.cs]: Why don't you change "1" here?

File: SeaBattleLibrary/ShipKillingValidation.cs [hunk 2]
```diff
@@ -138,9 +138,9 @@ public static void ShipKilling(int Index, int Letter)
             }
             else
             {
-                if (BattleShip.BotField[Index + 1, Letter] != 1 & BattleShip.BotField[Index - 1, Letter] != 1)
+                if (BattleShip.BotField[Index + 1, Letter] != Cells.Ship & BattleShip.BotField[Index - 1, Letter] != Cells.Ship)
                 {
-                    if (BattleShip.BotField[Index + 1, Letter] == 0 & BattleShip.BotField[Index - 1, Letter] == 0)
+                    if (BattleShip.BotField[Index + 1, Letter] == Cells.Untoched & BattleShip.BotField[Index - 1, Letter] == Cells.Untoched)
```
Comment (HeorhiSemianeniaiTechArt) [inline SeaBattleLibrary/ShipKillingValidation.cs]: Please rename "Untoched" to correct name. My bad(

File: SeaBattleLibrary/ShipPlacementValidation.cs (hunk 1 above)
Comment (ashurakova) [reply SeaBattleLibrary/ShipPlacementValidation.cs]: Fixed

File: SeaBattleLibrary/ShipKillingValidation.cs (hunk 2 above)
Comment (ashurakova) [reply SeaBattleLibrary/ShipKillingValidation.cs]: Fixed

**Pull request merged**